gui/CameraView.cxx

```diff
@@ -666,13 +683,21 @@ void CameraView::addLandmark(
 
 //-----------------------------------------------------------------------------
 void CameraView::addResidual(
-  kwiver::vital::track_id_t id, double x1, double y1, double x2, double y2)
+  kwiver::vital::track_id_t id, double x1, double y1, double x2, double y2,
+  bool inlier)
 {
   QTE_D();
 
   Q_UNUSED(id)
 
-  d->residuals.addSegment(x1, y1, -0.2, x2, y2, -0.2);
+  if (inlier)
+  {
+    d->residuals_inlier.addSegment(x1, y1, -0.2, x2, y2, -0.2);
```

Comment:
Would there be any benefit assigning these different (from each other) z-orders? (The answer may be "no", but something to think about if you hadn't already.)